files/initramfs-tools/union-mount.j2

```diff
@@ -17,8 +17,9 @@ set_tmpfs_log_partition_size()
 
   # NOTE: certain platforms, when reaching initramfs stage, have a small
   #       limit of mounting tmpfs partition, potentially due to amount
-  #       of RAM available in this stage. e.g. Arista 7050 QX32
+  #       of RAM available in this stage. e.g. Arista 70[56]0 [CQ]X32[S]
```

Comment:
Comment is a bit confusing, because it can be read as though there there are 8 devices: 7050 CX32, 7050 CX32S, 7050 QX32, 7050 QX32S, 7060 CX32, 7060 CX32S, 7060 QX32, 7060 QX32S. Can you separate the 7050 and 7060, e.g., 

`#       of RAM available in this stage. e.g. Arista 7050 QX32[S], Arista 7060 CX32S`

Reply:
done